lib/helpers.js

```diff
@@ -214,17 +214,15 @@ async function getApkanalyzerForOs (sysHelpers) {
  * do not work to detect lock status. Instead, keyguard preferences helps to detect the lock condition.
  * Some devices such as Android TV do not have keyguard, so we should keep
  * screen condition as this primary method.
+ * `mIsShowing` and `mInputRestricted` are `true` in lock condition.
+ * Both `false` is unlock condition.
  *
  * @param {string} dumpsys - The output of dumpsys window command.
  * @return {boolean} True if lock screen is showing.
  */
 function isShowingLockscreen (dumpsys) {
-  if (/(mShowingLockscreen=true|mDreamingLockscreen=true)/gi.test(dumpsys)) {
-    return true;
-  }
-  // Fallback to KeyguardServiceDelegate preference.
-  // mIsShowing and mInputRestricted are true in lock condition.
-  return dumpsys.includes('mIsShowing=false', 'mInputRestricted=false');
+  return _.some(['mShowingLockscreen=true', 'mDreamingLockscreen=true'], (x) => dumpsys.includes(x))
+  || _.every(['mIsShowing=false', 'mInputRestricted=false'], (x) => dumpsys.includes(x));
```

Comment:
Please add an indentation here to enhance the readability. The comment also makes sense